package.json

```diff
@@ -146,7 +146,7 @@
     "ts-loader": "^2.0.3",
     "tvm": "^0.8.14",
     "typedoc": "https://github.com/coveord/typedoc",
-    "typescript": "2.8.1",
+    "typescript": "2.9.2",
```

Comment:
🤔 ?

Updating is fine, but please do so in a separate story if you need to.

Reply:
It also seems it breaks the file definition validation on older version of typescript (1.8)

Reply:
I did not end up needing to update. I reverted this change, thanks.